demisto_sdk/commands/format/update_readme.py

```diff
@@ -0,0 +1,135 @@
+from demisto_sdk.commands.format.update_generic import BaseUpdate
+
+
+class ReadmeFormat(BaseUpdate):
+    """ReadmeFormat class is designed to update integration and pack README file according to Demisto's convention.
+
+        Attributes:
+            input (str): the path to the file we are updating at the moment.
+            output_file (str): the desired file name to save the updated version to.
+    """
+
+    def __init__(self,
+                 input: str = '',
+                 output: str = '',
+                 path: str = '',
+                 no_validate: bool = False,
+                 verbose: bool = False,
+                 **kwargs):
+        super().__init__(input=input, output=output, path=path, no_validate=no_validate,
+                         verbose=verbose, **kwargs)
+        with open(self.source_file, 'r') as f:
+            self.readme_content = f.read()
+
+    def replace_url_in_content(self, relative_url: list, new_url: str):
+        """Replace the relative url link with the new url in README."""
+
+        # md link
+        if '[' in relative_url[0]:
+            old_link = relative_url[0] + '(' + relative_url[1] + ')'
+            new_link = relative_url[0] + '(' + new_url + ')'
+        # href link
+        else:
+            old_link = relative_url[0]
+            new_link = str.replace(relative_url[0], relative_url[1], new_url, 1)
+
+        self.readme_content = str.replace(self.readme_content, old_link, new_link, 1)
+        click.secho(f'Replaced {relative_url[1]} with {new_url}')
+
+    def get_relative_urls(self) -> List[list]:
+        """
+        Find all relative urls (md link and href links_ in README.
+        Returns: a regex list of urls.
+
+        """
+        relative_urls = re.findall(RELATIVE_MARKDOWN_URL_REGEX, self.readme_content,
+                                   re.IGNORECASE | re.MULTILINE)
+        relative_urls += re.findall(RELATIVE_HREF_URL_REGEX, self.readme_content,
+                                    re.IGNORECASE | re.MULTILINE)
+        relative_urls = [url for url in relative_urls if url[1]]
+        return relative_urls
+
+    def get_new_url_from_user(self, url: list) -> Optional[str]:
+        """ Given we found a relative url, the user has the following options-
+        1. Add https:// prefix.
+        2. Enter a new absolute address to replace the current.
+        3. Leave as is
+
+        Args:
+            url: relative url found in README
+
+        Returns: new url or None if we leave as is
+
+        """
+        old_url = str.strip(url[1])
+        new_address = None
+        if self.assume_yes:
+            new_address = 'https://' + old_url
+        else:
+            click.secho(f'Relative urls are not supported within README, Should https:// be added to the '
+                        f'following address? [Y/n]\n {url[1]}',
+                        fg='red')
+            user_answer = input()
+            if user_answer in ['y', 'Y', 'yes', 'Yes']:
```

Comment:
please check whether we have a function that searches for these values, I remember one vaguely

Reply:
me neither, feel free to create one under tools. No need to change anywhere (although it could be a nice)